site/docs/components/dialog/examples.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+<LivePreviewControls>
+
+    <LivePreview componentName="dialog" exampleName="Default" >
+      ### Default
+
+      The dialog displays with a title, content area, and action bar. The component is modal, meaning it prevents interaction with the rest of the UI while open and must be addressed by clicking one of the action buttons or dismissing with the close button. If the content of the dialog extends beyond the end of the screen, the `DialogContent` area will become scrollable.
+
+    </LivePreview>
+
+        <LivePreview componentName="dialog" exampleName="Accent" >
+    ### Accent
+
+    For default dialogs, you can pass the `accent` prop to `DialogTitle` which accepts a boolean to render an accent bar in the title of the dialog. You cannot render the accent bar when a status is also being rendered.
+    </LivePreview>
+
+    <LivePreview componentName="dialog" exampleName="CloseButton" >
+    ### Close Button
+
+You can also render the `DialogCloseButton` as a child of the `Dialog` component to render a close button in the top right corner of the dialog. You can pass the `onClick` prop like with any `Button` component to handle the close button click.
+
+</LivePreview>
+
+    <LivePreview componentName="dialog" exampleName="Info" >
+      ### Info
+
+      - Use the info dialog when you need to display general information.
```

Comment:
Wonder if we should change these to 'use the info status' (same for references of 'info dialog' below and the other statuses) because the status is dynamic, rather than it being a fixed 'info dialog'

Reply:
This sounds good to me. 

Looking at other components, Toast says 'use the info Toast' etc, whereas Tooltip refers to the statuses.

@mikearildbrown @ivan-calderon any thoughts?

Reply:
Agree, @libertymayc  made a good point. However, I was wondering if there is some content guidance about it. Is there a way to standardize these case scenarios across components @mikearildbrown, @sion-docs? Are there any rules when to refer as status or component variant? 

Darrin also brought this to my attention last week while he was working on Toast. Moreover, if we make these changes for Dialog; should we reflect the same changes for Toast? I'd like to hear some advise about it. TY!